dist/stat.py

```diff
@@ -3,15 +3,18 @@
 
 import re, string, sys, textwrap
 from dist import compare_srcfile, format_srcfile
+from operator import attrgetter
 
 # Read the source files.
 from stat_data import groups, dsrc_stats, connection_stats, conn_dsrc_stats, join_stats, \
     session_stats
 
 connection_statistics = connection_stats
 connection_statistics.extend(conn_dsrc_stats)
+connection_statistics = sorted(connection_statistics, key=attrgetter('desc'))
```

Comment:
Much better solution than what I had in my patch 👍

Reply:
For clarity and to avoid potential further confusion can we rename `connection_statistics` to `sorted_connection_stats`, similarly for the datasource ones. I feel like its misleading having both `connection_statistics` and `connection_stats` in one file.

Reply:
See cc5be56